xrally_kubernetes/tasks/validators.py

```diff
@@ -28,3 +28,31 @@ def validate(self, context, config, plugin_cls, plugin_cfg):
         if "kubernetes" not in context["platforms"]:
             self.fail("There is no specification for kubernetes platform in "
                       "selected environment.")
+
+
+@validation.configure(name="map_keys")
```

Comment:
It would be nice if you propose this validator to rally repo and use here `If rally.__version__ < (1.2):`

Reply:
Let add separate patch for "deprecating" this for rally >= 1.2 when upstream rally patch will be merged.